src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -569,6 +581,14 @@ private void OnNodeViewMouseLeave(object sender, MouseEventArgs e)
             {
                 PreviewControl.TransitionToState(PreviewControl.State.Hidden);
             }
+
+            //hide the node autocomplete marker
+            foreach (PortViewModel port in ViewModel.OutPorts)
+            {
+                //skip code blocks
+                if (port.PortName.Equals(">")) continue;
```

Comment:
So we have access to the node that owns the port, can we skip the whole thing based on that ?
And should this go to the beginning of this function since there are some returns above ?